docs/examples/MarrmotM01.ipynb

```diff
@@ -2,196 +2,142 @@
  "cells": [
```

Comment:
<p>Before the mat file was downloaded from GH in this nb, now its downloaded at https://ewatercycle.readthedocs.io/en/latest/system_setup.html#download-example-forcing . Can we add a link to download instructions?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/eWaterCycle/ewatercycle/pull/158/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>

Reply:
<p>it is added.</p>